test/cloudtest/test_disk.py

```diff
@@ -0,0 +1,58 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+from textwrap import dedent
+
+from materialize.cloudtest.app.materialize_application import MaterializeApplication
+
+
+def test_disk_source(mz: MaterializeApplication) -> None:
```

Comment:
Can you consider extra checks here that the proper directory is indeed created where expected and stuff is written in it? This test as currently written will pass even if the entire machinery is broken and nothing gets written to any disk for any reason

Reply:
now it does, thanks:

```
commit b61f1f1d8f1e85ca0dc42af670dbbedf116c8395
Author: Paul Hemberger <paul@materialize.com>
Date:   Wed Jul 5 17:03:13 2023 -0400

    add cloudtest assertions that scratch dir is used
```